orbbec_camera/src/ob_camera_node.cpp

```diff
@@ -987,7 +987,7 @@ void OBCameraNode::getParameters() {
     param_name = stream_name_[stream_index] + "_fps";
     setAndGetNodeParameter(fps_[stream_index], param_name, 0);
     param_name = "enable_" + stream_name_[stream_index];
-    if (stream_index == DEPTH) {
+    if (stream_index == DEPTH || stream_index == COLOR) {
```

Comment:
@yrh012 Did we do this change such that depth and color are the two streams that are always active?